src/mono/wasm/wasm.proj

```diff
@@ -0,0 +1,43 @@
+<Project DefaultTargets="Build">
+  <Import Project="Directory.Build.props" />
+  <Import Project="Directory.Build.targets" />
+
+  <UsingTask TaskName="WasmAppBuilder"
+      AssemblyFile="$(MonoProjectRoot)msbuild\WasmAppBuilder\bin\publish\WasmAppBuilder.dll"/>
+
+  <PropertyGroup>
+	<WasmPInvokeTablePath>$(MonoObjDir)wasm/pinvoke-table.h</WasmPInvokeTablePath>
```

Comment:
fix the spaces vs. tabs mismatch here and in the rest of the file to only use spaces.